### ISSUE TITLE
cannot import ratings

### ISSUE THREAD

**Issue by pompobit** [genderquery/netflix-migrate#19]
after the fix on #16 I can export ratings (thanks), but seems like the import functions is now broken.

This is what I get:

```
TypeError: netflix.setThumbRating is not a function
    at /usr/lib/node_modules/netflix-migrate/main.js:160:23
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Error: Could not set rating for undefined. For more information, please see previous log statements.
    at /usr/lib/node_modules/netflix-migrate/main.js:166:13
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by LBBO** [comment]
That is actually exactly what I fixed in a1326ff, just before forgetting to `npm publish`... I should probably setup a step in the CI that automatically pushes changes on master or something, because I ALWAYS forget  to do it myself.

**Comment by pompobit** [comment]
ahah, well, still better than a new bug ;)
Thank you

**Comment by LBBO** [comment]
Yup. FYI: it should be published by now. Sorry for the trouble!

**Comment by LBBO** [comment]
TL;DR: Yes and no but, even with a fresh install from npm, I have no problem updating my ratings.

What currently works is a complete export of your ratings, wether they are stars or thumbs. When you import the data, you only get stars though. If a rating used to be a thumb up, it is now a two star rating (if I'm not mistaken).

I already have code that I believe works and that enables importing thumb ratings (without you having to update your export). However, I haven't had enough time to test it and so I haven't published it yet. A part of that code was what I had to remove in a1326ff because I had committed it by accident.

Just now, I tested updating ratings with a completely fresh install from npm and it seems to have worked perfectly. I only tried updating two ratings for speed purposes, but I see no reason why it shouldn't work for more ratings.

Could you maybe re-check all your data (especially the Profile name) and make sure to hard-reload the ratings page after updating them (CTRL / Cmd + Shift + R in most browsers)? If you still have the problem afterwards, this might become way more complicated.

**Comment by pompobit** [comment]
I'm not sure where stars come from, googling it I see that in the past Netflix used a star system further replaced with the thumbs up/down system but since I have an account I only had thumbs to select.
I tried several times specifying the profile name both with the command line parameter and with the prompt, using all lowercase for the profile name, replacing 2 stars in the json with 5 stars...
None of them worked, on the netflix webpage I still see every title without the thumbs selected.

If I try to export the rating from the account where I imported them, the json seems updated and I get the right number of the stars like the ones where I first exported them, but that's all, the new netflix account seems still fresh new from their site.

**Comment by pompobit** [comment]
thank you for your explanation, that makes sense.
In the meanwhile can I suggest you to change the behaviour to assign 2 stars to thumb up with a 5 stars conversion? I think it makes more sense, if the Netflix algorithms still takes stars in account I doubt that 2 stars count as a positive review ;)
That's just a suggestion, I already edited the exported rating searching and replacing the 2 stars with 5.

Anyway I'll keep my stars for who deserves and display them, for example this repository on github ;)
